pipelines/foreman_pipeline.yml

```diff
@@ -0,0 +1,29 @@
+- hosts: localhost
+  vars_files:
+    - vars/foreman_base.yml
+  roles:
+    - forklift
+
+- hosts:
+  - pipeline-foreman-{{ foreman_version }}-{{ foreman_os }}
+  become: yes
+  vars_files:
+    - vars/foreman_base.yml
+    - vars/foreman_base_staging.yml
+  roles:
+    - disable_firewall
+    - foreman
+
+- hosts: pipeline-foreman-{{ foreman_version }}-{{ foreman_os }}
```

Comment:
Do you want to merge this into the previous play since it's on the same host or do you intend to add a proxy at some point?